src/Microsoft.Azure.WebJobs.ServiceBus/ServiceBusTopicAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.ServiceBus.Messaging;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <inherit/>
+    [AttributeUsage(AttributeTargets.Parameter)]
+    public sealed class ServiceBusTopicAttribute : ServiceBusAttribute
```

Comment:
That's what I was envisioning, I prefer having a separate attribute `ServiceBusTopic` to having an `IsTopic` flag on the main attribute constructor.

I think I should also add a `ServiceBusQueue` attribute to make things more symmetrical.

Does that sound reasonable?